lib/widgets/compose_box.dart

```diff
@@ -160,10 +160,17 @@ class ComposeTopicController extends ComposeController<TopicValidationError> {
     return trimmed.isEmpty ? kNoTopicTopic : trimmed;
   }
 
+  /// Whether [textNormalized] would fail a mandatory-topics check
+  /// (see [mandatory]).
+  ///
+  /// The term "Vacuous" draws distinction from [String.isEmpty], in the sense
+  /// that certain strings are empty but also indicate the absence of a topic.
+  bool get isTopicVacuous => textNormalized == kNoTopicTopic;
```

Comment:
```suggestion
  /// The term "Vacuous" draws distinction from [String.isEmpty], in the sense
  /// that certain strings are not empty but also indicate the absence of a topic.
  bool get isTopicVacuous => textNormalized == kNoTopicTopic;
```

?  Or am I misunderstanding what you mean?